.eslintrc

```diff
@@ -0,0 +1,68 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings"
+  ],
+  "plugins": [
+    "react"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true,
+    "mocha": true
+  },
+  "rules": {
```

Comment:
Instead of using your rules (which is quite personnal in my opinion :) ), I think we should stick to a standard: can you add the AirBnB eslintrc configuration (https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb) instead of your own?